cji_smoke_test.sh

```diff
@@ -55,7 +56,14 @@ fi
 echo "Running: docker pull ${MC_IMAGE}"
 docker pull ${MC_IMAGE}
 
-CJI_NAME="$COMPONENT_NAME"
+
+# check for $IQE_CJI_NAME. If exists use it for CJI_NAME. If DNE use $COMPONENT_NAME for CJI_NAME
+if [[ -z $IQE_CJI_NAME ]]; then
+    CJI_NAME="$COMPONENT_NAME"
+else
+    CJI_NAME="$IQE_CJI_NAME"
+fi
+
```

Comment:
```suggestion
if [[ -z "$IQE_CJI_NAME" ]]; then
    CJI_NAME="$COMPONENT_NAME"
else
    CJI_NAME="$IQE_CJI_NAME"
fi

```

Reply:
sure thing. Do we also need them around $IQE_CJI_TIMEOUT ?

Reply:
Quotes added.

Reply:
Don't worry for now, let's focus on this change. This project sure could use a shellcheck review. [I did try once](https://github.com/RedHatInsights/bonfire/pull/297/files) but there were too many changes and had to revert. One of this days I'll get it done

Reply:
Quotes added.